sdcm/sct_config.py

```diff
@@ -365,7 +371,7 @@ class SCTConfiguration(dict):
 
         dict(name="authenticator", env="SCT_AUTHENTICATOR", type=str,
              help="which authenticator scylla will use AllowAllAuthenticator/PasswordAuthenticator",
-             choices=("PasswordAuthenticator", "AllowAllAuthenticator"),
+             choices=("PasswordAuthenticator", "AllowAllAuthenticator", "com.scylladb.auth.SaslauthdAuthenticator"),
```

Comment:
like the other 2 previous ones, why does it (and only it) needs the `com.scylladb.auth`?

Reply:
> like the other 2 previous ones, why does it (and only it) needs the `com.scylladb.auth`?

The 2 previous were from Cassandra.  The last one is the formal namespace, the prefix is necessary.